src/main/resources/templates/db_scripts/budget_encumbrances_rollover.sql

```diff
@@ -125,13 +125,22 @@ CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.rollover_order(_order_id
         SELECT public.uuid_generate_v5(public.uuid_nil(), concat('BER1', tr.id)), tr.jsonb - 'id' || jsonb_build_object
             (
                 'fiscalYearId', _rollover_record->>'toFiscalYearId',
-                'amount', ${myuniversity}_${mymodule}.calculate_planned_encumbrance_amount(tr.jsonb, _rollover_record, true),
+                'amount', CASE WHEN (tr.jsonb->'encumbrance'->>'reEncumber')::boolean
```

Comment:
Could we move this test to `calculate_planned_encumbrance_amount()` ? This function is used in many places and it looks like it should always return 0 when `reEncumber` is false.